lib/offline/storage.js

```diff
@@ -550,32 +562,21 @@ shaka.offline.Storage = class {
    * It is up to the caller to ensure that this method is not called
    * concurrently on the same manifest.
    *
+   * @param {shaka.extern.StorageCell} storage
    * @param {number} manifestId
+   * @param {!shaka.extern.ManifestDB} manifestDB
```

Comment:
The JSDoc needs to be updated again, now that this no longer loads the manifest.

Reply:
No, this is accurate.  I added the parameter.  Before, it was loaded inside the method.  Now, it is provided by the caller, who already had a copy.  The caller's copy is updated in-place as well as stored in the DB.

Reply:
Oh, wait, I see what you mean now.  Yes, you're right.  I'll take care of that.

Reply:
Done!

Reply:
I wish Github let me comment on lines that the PR didn't change. It'd make this kind of thing so much clearer.